Uno.Gallery/Entities/Sample.cs

```diff
@@ -40,7 +40,7 @@ private object CreateData(Type dataType)
 			}
 			catch (Exception e)
 			{
-				this.Log().Error($"Failed to initialize data for `{ViewType.Name}`:", e);
+				this.Log().Error($"Failed to initialize data for `{ViewType.Name}`. dataType: {dataType}. Exception: {e}");
```

Comment:
just for the sake of it..:
```suggestion
				this.Log().Error($"Failed to initialize data for `{ViewType?.Name}`. dataType: {dataType?.Name}. Exception: {e}");
```

Reply:
Early in the method we explictly did a null check already